src/storage/DeleteEdgesProcessor.cpp

```diff
@@ -0,0 +1,43 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+#include "storage/DeleteEdgesProcessor.h"
+#include <algorithm>
+#include <limits>
+#include "base/NebulaKeyUtils.h"
+
+namespace nebula {
+namespace storage {
+
+void DeleteEdgesProcessor::process(const cpp2::DeleteEdgesRequest& req) {
+    auto spaceId = req.get_space_id();
+    std::for_each(req.parts.begin(), req.parts.end(), [&](auto &partEdges) {
+        callingNum_ += partEdges.second.size();
+    });
+    CHECK_NOTNULL(kvstore_);
+
+    std::for_each(req.parts.begin(), req.parts.end(), [&](auto &partEdges) {
+        auto partId = partEdges.first;
+        std::for_each(partEdges.second.begin(), partEdges.second.end(), [&](auto &edgeKey) {
+            auto start = NebulaKeyUtils::edgeKey(partId,
+                                                 edgeKey.src,
+                                                 edgeKey.edge_type,
+                                                 edgeKey.ranking,
+                                                 edgeKey.dst,
+                                                 0);
```

Comment:
After #854 merged, the timestamp has been converted to big endian. I am not pretty sure if we can delete all edge from 0 to in64_max. @dangleptr @darionyaphet, WDYT?

Reply:
IMO, Big-endian is right for this operation, not Little-endian.